spec/factories/factories.rb

```diff
@@ -332,6 +333,28 @@
     min_label nil
   end
 
+  factory :answer5, class: Answer do
```

Comment:
I don't think you need to add two new factories here:
- `answer` factory is in [this file](https://github.com/expertiza/expertiza/blob/master/spec/factories/quiz_factory.rb#L77). And I understand it is difficult to find it.
- there has already had a `question` factory in this file.